feedservice/handler/handler.go

```diff
@@ -0,0 +1,49 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/greenbone/eulabeia/connection"
+	"github.com/greenbone/eulabeia/messages"
+	"github.com/greenbone/eulabeia/models"
+)
+
+type FeedHandler struct {
+	GetVt   func(oids string) (models.VT, error) // Function to get VTs from feedservice
+	Context string
+	ID      string
+}
+
+func (handler FeedHandler) On(topic string, message []byte) (*connection.SendResponse, error) {
+	var msg models.GetVT
+	if err := json.Unmarshal(message, &msg); err != nil {
+		return nil, err
+	}
+	mt, err := messages.ParseMessageType(msg.Type)
+	if err != nil {
+		return nil, err
+	}
+	if mt.Aggregate == "feed" {
+		switch mt.Function {
+		case "get":
+			vt, err := handler.GetVt(msg.ID)
+			if err != nil {
+				return nil, err
+			}
+
+			return &connection.SendResponse{
+				MSG: models.SendVT{
+					Identifier: messages.Identifier{
+						Message: messages.NewMessage("vts.feed", "", msg.GroupID),
```

Comment:
should the message name be `got.vts` or `got.feed` on a get request?

Reply:
I will change the method to `GotVT` and rename the aggregate to vt instead of feed, as it is more accurate.